src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -3,38 +3,6 @@ imports:
 
     - { resource: "@SyliusAdminBundle/Resources/config/app/events.yaml" }
     - { resource: "@SyliusAdminBundle/Resources/config/app/sylius/sylius_mailer.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/address_log_entry.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/admin_user.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/catalog_promotion.yaml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/channel.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/country.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/currency.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/customer.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/customer_group.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/customer_order.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/exchange_rate.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/inventory.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/locale.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/order.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/payment.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/payment_method.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_association_type.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_attribute.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_from_taxon.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_option.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_review.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_variant.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/product_variant_with_catalog_promotion.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/promotion.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/promotion_coupon.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/shipment.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/shipping_category.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/shipping_method.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/tax_category.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/tax_rate.yml" }
-    - { resource: "@SyliusAdminBundle/Resources/config/grids/taxon.yml" }
```

Comment:
This grid has been removed and not replaced because it has not been in used  since before 1.0.0. See  https://github.com/Sylius/Sylius/commit/95b1cbc000eab9b9e06b495a34afff82e942efbc#diff-8b388c80c5f84373ad2d744ae1ead4cbf0187a798596191d65cece1b48315e35R7